src/category_theory/limits/shapes/terminal.lean

```diff
@@ -227,4 +227,27 @@ is_colimit.cocone_point_unique_up_to_iso
 
 end
 
+section comparison
+variables {C} {D : Type u₂} [category.{v} D] (G : C ⥤ D)
+
+/--
+The comparison morphism from the image of a terminal object to the terminal object in the target
+category.
+TODO: Show this is an isomorphism if and only if `G` preserves terminal objects.
```

Comment:
Do we want these `TODO`s to show up in the docstrings, or only in the source code? In the latter case, please move them a line lower.

Reply:
Oops, thanks for this, I'll amend! I should have a PR coming soon resolving these anyway